tests/test_configs.py

```diff
@@ -21,6 +21,7 @@ def test_enhance_paths():
     with pytest.raises(ValueError):
         pineko.configs.enhance_paths(test_configs)
     test_configs["paths"]["operator_cards"] = pathlib.Path("my/ope/cards/")
+    test_configs["paths"]["operator_card_template_name"] = "_template.yaml"
```

Comment:
you should check he leaves this field alone ( a) it's sensible and b) you get another line coverage ;-) )